### ISSUE TITLE
Get started buttons

### ISSUE THREAD

**Issue by claireristow** [Lullabot/lb-ebook-poc#37]
This is a Level AAA violation so low priority but it couldn't hurt to add an aria-label to the get started buttons to provide additional context for screen reader users.

For example, the aria-label could be "Read the {{ ebook_title }} ebook." Or something along those lines. 
https://ebooks.lullabot.com/ebooks/my-sample-ebook/cover/ 

![Screenshot 2024-07-17 at 11 07 33 AM](https://github.com/user-attachments/assets/e51fecd1-24cd-4834-956e-fb58083115a8)
